### PR TITLE
[DOCS][8.3] Updates "Endpoint Security" to "Endpoint and Cloud Security" screenshots

### LINKED ISSUE #2032: [DOCS] Update Endpoint Security screenshots
Related issue: #2025 

We need to update all .pngs and/or gifs that reference the former Endpoint Security integration and update it with the new one: 

image

Topics: 

-  [Install Endpoint Security integration](https://www.elastic.co/guide/en/security/current/install-endpoint.html)
     - [x] security-integration.png
     - [x] add-elastic-endpoint-security.png
     - [x] add-agent.png
     - [x] endpoint-configuration.png

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2032 by updating the screenshots in the [Install Endpoint topic](https://www.elastic.co/guide/en/security/current/install-endpoint.html) to reflect the integration's new name. 

Also updates hard-coded references to the old name in the page's text, including its title. 

[Preview](https://security-docs_2075.docs-preview.app.elstc.co/guide/en/security/master/install-endpoint.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
LGTM! I also checked the outputs to make sure the attributes rendered correctly and didn't find any issues. Thanks for taking care of this, @benironside !

File: docs/getting-started/install-endpoint.asciidoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [[install-endpoint]]
 [role="xpack"]
-= Configure and install the Endpoint Security integration
+= Configure and install the {endpoint-cloud-sec} integration
 
-Like other Elastic integrations, Endpoint Security can be integrated into the Elastic Agent through {fleet-guide}/fleet-overview.html[{fleet}]. Upon configuration, the integration allows the Elastic Agent to monitor for events on your host and send data to the {security-app}.
+Like other Elastic integrations, {endpoint-cloud-sec} can be integrated into the Elastic Agent through {fleet-guide}/fleet-overview.html[{fleet}]. Upon configuration, the integration allows the Elastic Agent to monitor for events on your host and send data to the {security-app}.
```
Comment (joepeeples) [inline docs/getting-started/install-endpoint.asciidoc:5]: While we're at it, might as well fix these references to Elastic Agent:
```suggestion
Like other Elastic integrations, {endpoint-cloud-sec} can be integrated into the {agent} through {fleet-guide}/fleet-overview.html[{fleet}]. Upon configuration, the integration allows the {agent} to monitor for events on your host and send data to the {security-app}.
```
```suggestion
Like other Elastic integrations, {endpoint-cloud-sec} can be integrated into the Elastic Agent through {fleet-guide}/fleet-overview.html[{fleet}]. Upon configuration, the integration allows the Elastic Agent to monitor for events on your host and send data to the {security-app}.
```

**Review by joepeeples — APPROVED** [review]
LGTM! Also happened to notice a couple references to "Elastic Agent" that could be changed to {agent}.

File: docs/getting-started/install-endpoint.asciidoc [hunk 2]
```diff
@@ -63,14 +63,14 @@ If you have upgraded to an {stack} version that includes {fleet-server} 7.13.0 o
...
-image::images/install-endpoint/add-agent.png[Add agent flyout on the Fleet page.]
+image::images/install-endpoint/endpoint-cloud-sec-add-agent.png[Add agent flyout on the Fleet page.]
 
 . Select an agent policy for the {agent}. You can select an existing policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
 +
 The selected agent policy should include {endpoint-sec}.
 +
 [role="screenshot"]
-image::images/install-endpoint/endpoint-configuration.png[Add agent flyout with Endpoint Security integration highlighted.,575]
+image::images/install-endpoint/endpoint-cloud-sec-add-agent-detail.png[Add agent flyout with Endpoint Security integration highlighted.,575]
```
Comment (joepeeples) [inline docs/getting-started/install-endpoint.asciidoc:73]:
```suggestion
image::images/install-endpoint/endpoint-cloud-sec-add-agent-detail.png[Add agent flyout with Endpoint and Cloud Security integration highlighted.,575]
```

**Review by jmikell821 — APPROVED** [review]
LGTM! 🎉

**Comment by muskangulati-qasource** [comment]
Hi @jmikell821,

We tested this PR and found a few issues in the text used for integrations. We have shared the screenshots for the same below.

PR1
PR2

We will validate this ticket once again when the changes are reflected.

Thanks!

**Comment by benironside** [comment]
Hi @muskangulati-qasource, thanks for your helpful review. The issues you found in this PR have been fixed.

**Comment by muskangulati-qasource** [comment]
Hi @benironside,

We have validated these preview links for the respected changes and all changes are available and good to go.

Hence, we are marking this ticket as 'QA Validated'.

Thanks!

**Pull request merged**